.github/workflows/packages.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
       run: scripts\build_windows.cmd
       shell: cmd
     - name: set up github token
-      run: echo 'GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}' >> $GITHUB_ENV
+      run: echo GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} >> $GITHUB_ENV
```

Comment:
Right, the single quotes on windows
![image](https://user-images.githubusercontent.com/5367102/95891832-47706f00-0d8e-11eb-98b9-7188f93ac61c.png)